### PR TITLE
Add content ES runbook

### PR DESCRIPTION
**Pull request by MiroslavGatsanoga**
# Description

## What

Add a runbook for Content Elasticsearch Cluster.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181)

## Anything, in particular, you'd like to highlight to reviewers

I have skipped some sections of the runbook like **Primary URL**, **Monitoring** and **Second Line Troubleshooting**, because I was not sure what is the appropriate information for them. I am open for any suggestions regarding those. 

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [x] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/upp-content-es-runbook.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+content
+
+## Known About By
+
+- hristo.georgiev
+- robert.marinov
+- elina.kaneva
+- georgi.ivanov
+- tsvetan.dimitrov
+- dimitar.terziev
```
Comment (ekaneva) [inline runbooks/upp-content-es-runbook.md:32]: Add more people here. E.g. I always add Platform health team. We aim to have ~10 people in known about.

File: runbooks/upp-content-es-runbook.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+There are two regional Amazon Elasticsearch clusters, one in eu-west-1 and one in us-east-1. Each regional cluster contains 4 data nodes and 3 dedicated master nodes.
```
Comment (ekaneva) [inline runbooks/upp-content-es-runbook.md:40]: @gotha , can you please check if is this piece of information correct?

File: runbooks/upp-content-es-runbook.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+PartiallyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+The service is part of both Delivery clusters. The failover guide for Delivery clusters can be found here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```
Comment (ekaneva) [inline runbooks/upp-content-es-runbook.md:64]: Put the `<` `>` around all URLs:
<https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>

File: runbooks/upp-content-es-runbook.md (hunk 1 above)
Comment (MiroslavGatsanoga) [reply runbooks/upp-content-es-runbook.md:32]: Ok, I did not know that we aim for ~10 people per system, I will add PH team as well.

**Pull request merged**